string  ctrl_type
float64  time
float64  position
float64  velocity
float64  commanded_effort
float64  measured_effort
float64  filtered_velocity
float64  filtered_effort
bool  max_effort_reached

